tests/e2e-pw/README.md

```diff
@@ -0,0 +1,88 @@
+
+## FAQs
+
+**How do I wait for a page or element to load?**
+
+Since [Playwright automatically waits](https://playwright.dev/docs/actionability) for elements to be present in the page before interacting with them, you probably don't need to explicitly wait for elements to load. For example, all of the following locators will automatically wait for the element to be present and stable before asserting or interacting with it:
+
+```ts
+await expect( page.getByRole( 'heading', { name: 'Sign up' } ) ).toBeVisible();
+await page.getByRole( 'checkbox', { name: 'Subscribe' } ).check();
+await page.getByRole( 'button', { name: /submit/i } ).click();
+```
+
+In some cases, you may need to wait for the page to reach a certain load state before interacting with it. You can use `await page.waitForLoadState( 'domcontentloaded' );` to wait for the page to finish loading.
+
+**What is the best way to target elements in the page?**
+
+Prefer the use of [user-facing attribute locators or test-ids](https://playwright.dev/docs/locators#locating-elements) to target elements in the page. This will make the tests more resilient to changes to implementation details, such as class names.
+
+```ts
+// Prefer locating by role, label, text, or test id when possible. See https://playwright.dev/docs/locators
+await page.getByRole( 'button', { name: 'All deposits' } ).click();
+await page.getByLabel( 'Select a deposit status' ).selectOption( 'Pending' );
+await expect( page.getByText( 'Order received' ) ).toBeVisible();
+await page.getByTestId( 'accept-dispute-button' ).click();
+
+// Use CSS selectors as a last resort
+await page.locator( 'button.components-button.is-secondary' );
+```
+
+**How do I create a visual regression test?**
+
+Visual regression tests are captured by the [`toHaveScreenshot()` function](https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-screenshot-2). This function takes a screenshot of a page or element and compares it to a reference image. If the images are different, the test will fail.
+
+```ts
+await expect( page ).toHaveScreenshot();
+
+await expect(
+	page.getByRole( 'button', { name: 'All deposits' } )
+).toHaveScreenshot();
+```
+
+**How can I act as shopper or merchant in a test?**
+
+1. A test suite (`describe`) can act as `shopper` or `merchant` by using the helper function `useShopper` or `useMerchant` from `tests/e2e-pw/utils/helpers.ts`.
+2. To switch between `shopper` and `merchant` role, use the `getShopper` and `getMerchant` function:
+
+```ts
+import { getShopper, getMerchant } from './utils/helpers';
+
+test( 'should do things as shopper and merchant', async ( { browser } ) => {
+	const { shopperPage } = await getShopper( browser );
+	const { merchantPage } = await getMerchant( browser );
+
+	// do things as shopper
+	await shopperPage.goto( '/cart/' );
+
+	// do things as merchant
+	await merchantPage.goto( '/wp-admin/admin.php?page=wc-settings' );
+} );
+```
```

Comment:
While migrating the merchant disputed order notice test, I noticed the problem of switching between `shopper` and `merchant` within a test.

I added util functions to access `shopperPage` and `merchantPage`. What do we think about this pattern?